spec/models/loader/types.rb

```diff
@@ -0,0 +1,37 @@
+require 'spec_helper'
+
+describe Loader::Types do
+  context "Check user creation verification" do
+    def new_user()
```

Comment:
Typically this is done with a `let` block instead of a method:
```ruby
let(:new_user) {  Loader::Types::User.new(id: 'alice') }
```

Reply:
done